release/CMakeLists.txt

```diff
@@ -48,7 +48,7 @@ set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "${CPACK_PACKAGE_DESCRIPTION_SUMMARY}
 
 # Add `openssl` package to dependencies list. Used to generate SSL certificates.
 # We also depend on `python3` because we embed it in Memgraph.
-set(CPACK_DEBIAN_PACKAGE_DEPENDS "openssl (>= 1.1.0), python3 (>= 3.5.0), libstdc++6")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "openssl (>= 1.1.0), python3 (>= 3.5.0)")
```

Comment:
Add `logrotate` (audit logs) + `curl` (telemetry)

Reply:
Seems like they already get added.

Reply:
`CPACK` black magic, `curl` is added to the list, but `logrotate` is not for some reason...

Reply:
I'll skip adding new stuff for now, maybe during the next release (v2.10)